sheepdog/transactions/submission/transaction.py

```diff
@@ -134,7 +134,8 @@ def take_action(self):
         For the current project, attempt to transition all nodes from their
         current states to ``submitted``.
         """
-        self.assert_project_state()
+        if not self._config.get('IGNORE_PROJECT_STATE'):
```

Comment:
I would put this inside `self.assert_project_state()` so no new code will break the flag if someone forgets about it